src/utils.js

```diff
@@ -546,16 +546,18 @@ var hasOwn = function (objectToCheck, propertyToCheckFor) {
 exports.insertElement = function(elm, doc, target) {
   doc = doc || document;
   let elToAppend;
+  const head = doc.getElementsByTagName('head');
   if (target) {
     elToAppend = doc.getElementsByTagName(target);
   } else {
-    elToAppend = doc.getElementsByTagName('head');
+    elToAppend = head;
   }
   try {
     elToAppend = elToAppend.length ? elToAppend : doc.getElementsByTagName('body');
     if (elToAppend.length) {
       elToAppend = elToAppend[0];
-      elToAppend.insertBefore(elm, elToAppend.firstChild);
+      const refChild = head && head[0] === elToAppend ? null : elToAppend.firstChild;
```

Comment:
Won't this only insert it before the `firstChild`?

Reply:
The `null` makes it append after the last child weirdly.
https://developer.mozilla.org/en-US/docs/Web/API/Node/insertBefore#Syntax